types/algoliasearch/index.d.ts

```diff
@@ -1416,6 +1416,7 @@ declare namespace algoliasearch {
       | 'logs'
       | 'seeUnretrievableAttributes')[];
     validity: number;
+    indexes?: string[];
```

Comment:
Note: 
I didn't add the `indexes` because I'm not sure if ListApiKey returns it
cf https://www.algolia.com/doc/api-reference/api-methods/list-api-keys/

Reply:
@nunomaduro do you know?

Reply:
What is the question? `ListApiKey` returns a list of keys that may or not contain `indexes`.

Reply:
`indexes` is not listed in the documentation ListApiKey
so the question is, is this a documentation error or do we need a subset type ?

Reply:
Just tested, it's a documentation error, I'll open a PR.

Reply:
https://github.com/algolia/doc/pull/4043